v2/kafka-to-gcs/src/main/java/com/google/cloud/teleport/v2/transforms/WriteTransform.java

```diff
@@ -62,7 +68,8 @@ public POutput expand(PCollection<KafkaRecord<byte[], byte[]>> kafkaRecord) {
                     .setSchemaRegistryURL(options().getSchemaRegistryURL())
                     .setSchemaPath(options().getSchemaPath())
                     .setWindowDuration(options().getWindowDuration())
-                    .build());
+                    .build()
+                    .withBadRecordErrorHandlers(errorHandlers()));
```

Comment:
Calling `withX()` after `build()` seems like a strange pattern to me. Should `setErrorHandlers()` be part of the builder?

Reply:
Done